plainly-plugin/src/ui/components/form/UploadForm.tsx

```diff
@@ -75,6 +104,56 @@ export default function UploadForm() {
     }
   };
 
+  useEffect(() => {
+    const fetchProject = async (
+      key: string,
+      projectId: string,
+      revision: string,
+    ) => {
+      try {
+        const { data } = await get<Project>(
+          `/api/v2/projects/${projectId}`,
+          key,
+        );
+
+        const revisionHistory = data.revisionHistory;
+        const latestRevision = revisionHistory?.length || 0;
+
+        if (latestRevision.toString() !== revision) {
```

Comment:
* why to string? data loaded from the project XML should be transformed to number?
* but does not work anyway